include/aikido/planner/World.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef AIKIDO_PLANNER_WORLD_HPP_
+#define AIKIDO_PLANNER_WORLD_HPP_
+
+#include <string>
+#include <dart/dart.hpp>
+
+namespace aikido {
+namespace planner {
+
+class World
+{
+public:
+  /// Construct a kinematic World.
+  World(const std::string& name);
```

Comment:
If we are creating or cloning worlds inside of an optimizer or something, does it really make sense to need to generate names for all of them?  I think DART has some mechanisms for autogenerating IDs if names are not set, something like that might make sense here if this is just for semantic purposes.